Support netstandard2.0

neon-sunset opened the pull request:
A follow-up change to https://github.com/VSadov/NonBlocking/pull/13. There are platforms which could use this library but don't support `netstandard2.1` (e.g. older Unity versions, .NET FW).

neon-sunset committed:
Add netstandard2.0 TFM support

neon-sunset commented:
@VSadov Please review this change when/if you have time. Thanks!

VSadov commented:
I think this now needs rebasing, since it has changes from #13

VSadov commented:
I've rebased the PR onto master - to have only the new changes. Looks good otherwise. Thanks!!

neon-sunset commented:
> I've rebased the PR onto master - to have only the new changes. Looks good otherwise. Thanks!!

Right, sorry, I rebased it incorrectly and then some other things came up. Thank you for merging this.
